cas-server-support-jpa-ticket-registry/src/main/java/org/jasig/cas/ticket/registry/JpaTicketRegistry.java

```diff
@@ -206,15 +206,15 @@ private int deleteTicketGrantingTickets(final String ticketId) {
      * @param result the result
      * @return the int
      */
-    private static int countToInt(final Object result) {
-        final int intval;
+    private static long countToLong(final Object result) {
```

Comment:
this could be reduced to:
```
private static long countToLong(final Object result) {
    return ((Number) result).longValue();
}
```
no need to handle it differently for the various `Number` types.